ddl/column_change_test.go

```diff
@@ -127,17 +127,17 @@ func (s *testColumnChangeSuite) TestColumnChange(c *C) {
 	}
 	d.SetHook(tc)
 	defaultValue := int64(3)
-	job := testCreateColumn(c, ctx, d, s.dbInfo, tblInfo, "c3", &ast.ColumnPosition{Tp: ast.ColumnPositionNone}, defaultValue)
+	job := testCreateColumn(ctx, c, d, s.dbInfo, tblInfo, "c3", &ast.ColumnPosition{Tp: ast.ColumnPositionNone}, defaultValue)
 	c.Assert(errors.ErrorStack(checkErr), Equals, "")
 	testCheckJobDone(c, d, job, true)
 	mu.Lock()
 	tb := publicTable
 	mu.Unlock()
-	s.testColumnDrop(c, ctx, d, tb)
-	s.testAddColumnNoDefault(c, ctx, d, tblInfo)
+	s.testColumnDrop(ctx, c, d, tb)
+	s.testAddColumnNoDefault(ctx, c, d, tblInfo)
 }
 
-func (s *testColumnChangeSuite) testAddColumnNoDefault(c *C, ctx sessionctx.Context, d *ddl, tblInfo *model.TableInfo) {
+func (s *testColumnChangeSuite) testAddColumnNoDefault(ctx sessionctx.Context, c *C, d *ddl, tblInfo *model.TableInfo) {
```

Comment:
golint checks `context.Context` or `golang.org/x/context.Context`.
But it's  `ctx sessionctx.Context` here,  golint should not complain and the modify is unnecessary.